README.md

```diff
@@ -55,6 +54,14 @@ Die "gewünschteAnzahlVorschlaege" ist mit 2 vorbelegt. Die API kann bis zu 10 V
   },
 ```
 
+Um zwischen Test_Modus und Echt_Geschaeft zu wechseln, muss der Datenkontext angepasst werden.
```

Comment:
Bitte den Abschnitt _Metdaten_ ändern in :

- Testen der API
- Anpassen Verhalten der API beeinflussen